lib/charms/prometheus_pushgateway_k8s/v0/pushgateway.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details
+
+r"""Interface library for the Prometheus Pushgateway.
+
+This library wraps the relation endpoint using the `pushgwateway` interface
+and provides a Python API for sending metrics.
```

Comment:
```suggestion
and exposes an API for forwarding metrics to Prometheus.
```
'provides' is a bit overloaded in charm libs

Reply:
Improved, thanks!